packages/common/core-interfaces/package.json

```
@@ -137,8 +137,8 @@
 		"@fluid-tools/build-cli": "^0.58.3",
 		"@fluidframework/build-common": "^2.0.3",
 		"@fluidframework/build-tools": "^0.58.3",
-		"@fluidframework/core-interfaces-previous": "npm:@fluidframework/core-interfaces@2.71.0",
-		"@fluidframework/eslint-config-fluid": "^7.0.0",
+		"@fluidframework/core-interfaces-previous": "npm:@fluidframework/core-interfaces@2.70.0",
```

Bad conflict resolution given the recent release, looks like.
```suggestion
		"@fluidframework/core-interfaces-previous": "npm:@fluidframework/core-interfaces@2.71.0",
```